wiremock/mappings/ccd/callback_get_case_injectedData.json

```diff
@@ -0,0 +1,50 @@
+{
+  "request": {
+    "method": "POST",
+    "urlPathPattern": "/.*/callback_get_case_injectedData"
```

Comment:
This could just be

`"urlPath": "/callback_get_case_injectedData"`

Reply:
Fixed in [373cfe5](https://github.com/hmcts/ccd-test-stubs-service/pull/177/commits/373cfe5ec509189a82e7810ccd583254a1175e0e)